### PR TITLE
Polyhedron_demo: Make it possible to switch between projection modes

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR adds a switch in the MainWindow's View menu to make the projection matrix orthographic.

### CODE REVIEW

**Comment by sgiraudot** [comment]
For information, I was the one who asked for this feature (it can be very useful when dealing with objects were things are aligned, for example for selection).

I tested it locally, no error or warning and it works well.

**Comment by afabri** [comment]
Compiles and works well for Windows/VC++.

File: Polyhedron/demo/Polyhedron/MainWindow.ui [hunk 1]
```diff
@@ -676,6 +677,14 @@
     <string>Set Maximum  Text Items Displayed :</string>
    </property>
   </action>
+  <action name="actionSwitchProjection">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Orthographic/Perspective Projection</string>
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.ui]: If the action is checkable, its name shoudl be the name of the state once check: "Use orthographic projection instead".

Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.ui]: Its "text", not its name.

**Pull request merged**